code/FulpstationCode/Fulp_traitor_items/traitor_chaplain.dm

```diff
@@ -1,285 +1,646 @@
+				Bolt(current,origin,bolt_energy,bounces,bolt_range, user)
+				return
+			else
+				current.visible_message("<span class='warning'>[current] deflects the [src] with [S], remaining unharmed!</span>", "<span class='userdanger'>You deflect the [src] with [S], remaining unharmed!</span>")
+				return
+
+	if(current.anti_magic_check())
+		current.visible_message("<span class='warning'>[current] absorbs the [src], remaining unharmed!</span>", "<span class='userdanger'>You absorb the [src], remaining unharmed!</span>")
+	else
+		current.electrocute_act(bolt_energy,"Force Lightning",1,bolt_flags)
+		if(!iscarbon(current)) //So we affect borgs and simple mobs properly.
+			current.adjustFireLoss(bolt_energy)
+		spark_system.attach(target)
+		spark_system.start()
+
+	if(bounces < 1)
+		return
+
+	existing_targets += current
+
+	//if(current in existing_targets) //DEBUG
+	//	to_chat(world, "[current] in existing targets.") //DEBUG
+
+	var/list/possible_targets = list()
+	for(var/mob/living/M in view(bolt_range,current))
+		/*
+		if(user == M) //Don't ask me why I have to arrange the conditional gates in this way; it just does not work if it they're all in the same row with || separators.
+			//to_chat(world, "Target [M] skipped. [M] is caster: [user]") //DEBUG
+			continue*/
+		if (!los_check(current,M) )
+			//to_chat(world, "Target [M] skipped; LoS check to [current] failed.") //DEBUG
+			continue
+		else if (origin == M)
+			//to_chat(world, "Target [M] skipped; [M] is origin: [origin].") //DEBUG
+			continue
+		else if (M in existing_targets)
+			//to_chat(world, "Target [M] skipped; [M] is in existing targets.") //DEBUG
+			continue
+		else
+			possible_targets += M
+			//to_chat(world, "Target [M] added.") //DEBUG
+	if(!possible_targets.len)
+		return
+	var/mob/living/next = pick(possible_targets)
+	//to_chat(world, "Target [next]. Possible targets: [possible_targets]. Bolt Energy [bolt_energy]. User: [user]") //DEBUG
+	var/new_damage = bolt_energy - bolt_decay_per_bounce
+	if(next && new_damage >= 1)
+		Bolt(current,next,new_damage,bounces - 1,bolt_range - 1, user)
+
+#undef FORCELIGHTNING_REFLECT_MULTIPLIER
+
+// SITH LORD ENDS
+
+// JUSTICIAR'S FOLLOWER BEGINS
+
+// hey, this one is called traitor_chaplain. might as well, right? -SgtHunk
+
+// Traitor Kit. 20TC, Chaplain only.
+
+/obj/item/storage/box/syndicate/bundle_clock/PopulateContents()
+	new /obj/item/clothing/suit/space/clockwork(src)
+	new /obj/item/clothing/gloves/combat/clockwork(src)
+	new /obj/item/clothing/shoes/combat/swat/bronze(src)
+	new /obj/item/clothing/glasses/hud/clock(src)
+	new /obj/item/clothing/head/helmet/space/clockwork(src)
+	new /obj/item/book/granter/martial/justiciar_hand(src)
+
+// Whole set of Armor.
+
+/obj/item/clothing/head/helmet/space/clockwork
+	name = "brass hardsuit helmet"
+	desc = "Ancient helmet made out of brass. It seems to provide robust protection against most damage types. It glows with a golden, ancient and forgotten energy."
```

Comment:
might wanna specifically mention that it doesn't help much against lasers and disablers